book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+  attr_accessor :title, :author
+  attr_reader :rentals
+end
```

Comment:
- [ ] great job so far. please kindly ensure that a method for adding rentals should be present. note that here the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.`
eg
```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```

Reply:
Fixed.